snapshot-labs/sx-monorepo
# refactor(ui): migrate table to div

### Summary



Closes: #339 #330 #145

Migrate tables to div for pages:

- votes 
- delegates
- leaderboard

Migrating to div will fix a few issues with table

### How to test

1. Go to a proposal votes page
2. It should display data as before
3. The voting power background should display correctly on webkit engine (https://github.com/snapshot-labs/sx-monorepo/issues/145)
4. Go to the delegates and leaderboard page
5. It should display data as before
6. The 1px gap described in https://github.com/snapshot-labs/sx-monorepo/issues/330 should be fixed
